compiler/src/test/java/com/squareup/hephaestus/compiler/InterfaceMergerTest.kt

```diff
@@ -198,8 +194,9 @@ class InterfaceMergerTest(
     """
     ) {
       assertThat(exitCode).isEqualTo(COMPILATION_ERROR)
-      // Position to the class.
-      assertThat(messages).contains("Source.kt: (13, 11)")
+      // Position to the class. Unfortunately, a different error is reported that the class is
+      // missing an @Module annotation.
+      assertThat(messages).contains("Source.kt: (7, 7)")
```

Comment:
Wouldn't it be more useful to search for something more meaningful than the source lines? Might be annoying if you're hardcoding the line numbers and it changes on you.

Reply:
When I started writing these tests, the error messages slightly changed here and there. To me it was more important that the error is reported in the correct row and column and points to the offending piece of code. But at the same time I agree with you. That's a good opportunity to contribute :-)